tests/csrf_tests/tests.py

```diff
@@ -39,6 +39,7 @@ class CsrfViewMiddlewareTestMixin:
     """
 
     _csrf_id = _csrf_id_cookie = '1bcdefghij2bcdefghij3bcdefghij4bcdefghij5bcdefghij6bcdefghijABCD'
+    mw = CsrfViewMiddleware()
```

Comment:
Is this refactoring a needed part of this patch? Can in be done in a separate commit that precedes the bug fix? I see that at least the `process_request()` additions have to be in the fix.

Reply:
No, it is not necessarily needed, but the patch is small enough (and I am honestly to lazy to revert them and move them into an extra commit).